examples/typescript/platform-ref-aws/crossplane.ts

```diff
@@ -0,0 +1,77 @@
+import { App, Chart } from 'cdk8s';
+
+//import * as config from './crs/configuration.meta.pkg.crossplane.io';
+//import { Configuration } from 'crossplane-cdk';
+import { Configuration } from '../../../src';
```

Comment:
@eladb looks like theres still something we need to get this local import working, but have a workaround for now:

Cannot find module 'crossplane-cdk' from 'examples/typescript/platform-ref-aws/crossplane.js'

Reply:
I think the relative import is actually fine. It's not an uncommon practice in node modules to simply reference locally in examples. 

Let's leave it like this for now and get rid of the hack I did with the node modules?

Reply:
sounds good